.travis.yml

```diff
@@ -1,5 +1,10 @@
 ---
 language: ruby
-rvm: 2.1.2
-before_script: bundle exec appraisal install
-script: bundle exec appraisal rspec
+rvm:
+  - 2.1.8
```

Comment:
@aprescott sadly I believe that only pulls in the latest *pre-installed* version of Ruby, not the latest *available* version, and since Travis is usually very behind in its preinstalled versions, we'd always be way behind as well.